Gruntfile.js

```diff
@@ -453,6 +457,7 @@ module.exports = function (grunt) {
     grunt.registerTask('test-env-html', 'Build the phantom test html page.', function () {
         var buffer = fs.readFileSync('clients/web/test/testEnv.jadehtml');
         var globs = grunt.config('uglify.app.files')['clients/web/static/built/app.min.js'];
+        var dependencies = [];
```

Comment:
It looks like this only ever gets assigned as `[]`, so we can probably just remove the variable altogether and pass the empty array directly into `jsFilesUncovered`.